web/apps/serve.go

```diff
@@ -75,7 +75,7 @@ func handleIntent(c echo.Context, i *instance.Instance, slug, intentID string) {
 	if !allowed {
 		return
 	}
-	from := i.SubDomain(intent.Client).Host
+	from := i.SubDomain(intent.Client).String()
```

Comment:
Thanks @nono, it works better. Another problem is that when an intent is save in DB, the Client value is set to something like `io.cozy.apps/test-app`, the  `io.cozy.apps/` appears in the header and make the iframe loading fail, for Security reason.